sample_apps/dotnet/CrudAndSimpleIngestionExample.cs

```diff
@@ -335,6 +335,11 @@ public async Task WriteRecords()
                 };
                 WriteRecordsResponse response = await writeClient.WriteRecordsAsync(writeRecordsRequest);
                 Console.WriteLine($"Write records status code: {response.HttpStatusCode.ToString()}");
+            } 
+            catch (RejectedRecordsException e) {
+                foreach (RejectedRecord rr in e.RejectedRecords) {
```

Comment:
there is no equivalent of System.out.println("RejectedRecords: " + e); here.

Reply:
> there is no equivalent of System.out.println("RejectedRecords: " + e); here.

Added